src/theme/default/calendar.m.css

```diff
@@ -1,31 +1,38 @@
 @import './variables.css';
 
+/* The root class of the calendar */
 .root {
 	box-sizing: border-box;
 	display: inline-block;
 }
 
+/* Added to inactive dates (disabled or out of range) */
 .inactiveDate {
 	background-color: var(--disabled-color);
 }
 
+/* Added to dates outside of the allowed min/max dates */
 .outOfRange {
 }
 
+/* The currently selected date */
 .selectedDate {
 	background-color: var(--selected-background);
 	color: var(--selected-color);
 }
 
+/* Added to today's date */
 .todayDate {
 }
 
+/* Added to all dates */
 .date {
 	cursor: pointer;
 	padding: 10px 20px;
 	text-align: center;
 }
 
+/* Hover style for all dates */
 .date:hover {
```

Comment:
There's no need to comment on pseudo selectors as I do not believe they make it through to the generated docs.